neutrino.go

```diff
@@ -209,26 +209,36 @@ func (sp *ServerPeer) addressKnown(na *wire.NetAddress) bool {
 // the score is above the ban threshold, the peer will be banned and
 // disconnected.
 func (sp *ServerPeer) addBanScore(persistent, transient uint32, reason string) {
-	// No warning is logged and no score is calculated if banning is disabled.
+	// No warning is logged and no score is calculated if banning is
```

Comment:
Reason `addBanScore` isn't used anywhere atm? Just curious..

Reply:
Looks like Neutrino was able to respond to `getdata` messages from peers in a previous version and it used `addBanScore` there. That functionality was removed, but this remained.